app/grandchallenge/hanging_protocols/forms.py

```diff
@@ -160,7 +160,7 @@ def __init__(self, *args, **kwargs):
             reverse("documentation:detail", args=["viewer-content"]),
         )
 
-    def generate_view_content_example(self):

```

Comment:
The function became too long and triggered flake C901 error 
'ViewContentExampleMixin.generate_view_content_example' is too complex
So, I split it into smaller functions with specific tasks.